react/react1/week1/static-todo-list/src/App.css

```diff
@@ -0,0 +1,29 @@
+.App {
```

Comment:
Hey Julia, good job. you got to the essence of React quickly. Thumbs up for using separation of concerns 👍

In App.css you don't need to keep the default code, it can be confusing.

Reply:
Hi Gohar.
Thanks for the review. It is essential to start creating it just like in real-world.